packages/components/pie-spinner/src/index.ts

```diff
@@ -13,8 +20,30 @@ const componentSelector = 'pie-spinner';
  * @tagname pie-spinner
  */
 export class PieSpinner extends LitElement implements SpinnerProps {
+    @property({ type: Object })
+    public aria?: AriaProps;
+
+    @property()
+    @validPropertyValues(componentSelector, sizes, 'm')
+    public size?: SpinnerProps['size'] = 'm';
+
+    @property()
+    @validPropertyValues(componentSelector, variants, 'brand')
+    public variant?: SpinnerProps['variant'] = 'brand';
+
     render () {
-        return html`<h1 data-test-id="pie-spinner">Hello world!</h1>`;
+        const { variant, size, aria } = this;
+
+        return html`
+            <div 
+                data-test-id="pie-spinner"
+                class="c-spinner"
+                role="status"
```

Comment:
It might be worth adding some component tests for these two. That they exist for example.

Reply:
done 🤝